src/librados/RadosClient.cc

```diff
@@ -39,7 +39,6 @@
 // needed for static_cast
 #include "messages/PaxosServiceMessage.h"
 #include "messages/MPoolOpReply.h"
-#include "messages/MStatfsReply.h"
```

Comment:
i think it's safe to drop other `#include "messages/*` as well except for `MLog.h`.

Reply:
Good point, fixed!